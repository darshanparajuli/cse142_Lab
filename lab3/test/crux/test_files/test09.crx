// test block-level scope

func main() : void {
  var a : int;
  if (true) {
    var a: float;
    ::foo();
  }
}
