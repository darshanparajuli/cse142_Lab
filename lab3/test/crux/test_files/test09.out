Error parsing file.
ResolveSymbolError(7,7)[Could not find foo.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(main)
  Symbol(a)
    Symbol(a)


